seed.js

```diff
@@ -0,0 +1,80 @@
+const data = require('./seedData.json');
+
+const {
+    client,
+    createTables,
+    createCostume,
+    getAllCostumes,
+    getCostumeById,
+    updateCostume,
+    deleteCostumeById
+} = require(".");
+
+// Two costume entries
+const costumeOne = data[0];
+const costumeTwo = data[1];
+
+
+const seedDB = async () => {
+    console.log("begin seeding db");
+    console.log("creating tables");
+    await createTables();
+    console.log("successfully created tables");
+    const costumes = await getAllCostumes();
+    console.log("costumes:", costumes);
+    console.log("create costumes");
+    const hat = await createCostume(
+        costumeOne.name, 
+        costumeOne.category,
+        costumeOne.gender,
+        costumeOne.size,
+        costumeOne.type,
+        costumeOne.stock_count,
+        costumeOne.price
+    );
+    console.log("created ", hat);    
+    const costumesAgain = await getAllCostumes();
+    console.log(costumesAgain);
+    const shirt = await createCostume(
+        costumeTwo.name, 
+        costumeTwo.category,
+        costumeTwo.gender,
+        costumeTwo.size,
+        costumeTwo.type,
+        costumeTwo.stock_count,
+        costumeTwo.price
+    );
```

Comment:
Prefer using `data.forEach` or a `for` loop -that way seedData is extensible. Also to keep it DRY (repeat code = more room for bugs)

Reply:
Ooooo, I didn't even think to do that. Love it.